.github/workflows/build.yml

```diff
@@ -2,17 +2,18 @@ name: Build and push images
 on:
   push:
     branches:
-    - master
+      - master
     paths:
-    - "*"
-    - "!README.md"
-    - "!build.rb"
+      - "*"
+      - "!README.md"
+      - "!build.rb"
   schedule:
     # As well as running when we make changes we should run at least
     # every week in order to pick up new parent images and new versions of Snyk
     - cron:  "0 0 * * 0"
-  workflow_dispatch:
-
+  repository_dispatch:
```

Comment:
Question: Is there a reason why `workflow_dispatch` needs to be removed. I think it can stay to enable manual trigger.

Reply:
My bad. I thought we needed to have defined an [input](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs) to be able to use the `workflow_dispatch` - Reverting this change 🙏